### PR TITLE
config webpack

### PR DESCRIPTION
**Pull request by pixelpilot-dev**
- configured webpack with babel
- installed React
- DEV and PROD build commands

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  "name": "reactjs-gmp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start:dev": "cross-env NODE_ENV=development webpack serve --open",
+    "start:prod": "cross-env NODE_ENV=production webpack serve --open",
+    "build:dev": "cross-env NODE_ENV=development webpack",
+    "build:prod": "cross-env NODE_ENV=production webpack",
+    "lint:js": "./node_modules/.bin/eslint '**/*.js'",
```
Comment (Dmitry-White) [inline package.json]: Great idea about linting! Let's extend you solution further.
Consider adding "eslint" and "eslint:fix" commands in "scripts" section, which serve the purpose of "just checking" and "checking + autofixing" respectively. Same goes for the Prettier commands.
Then have "lint" and "lint:fix" that rely on the above commands to do both check and both fixes one after another.
Also consider adding separate ESlint and Prettier configs files to your project with some basic "airbnb" rules for JS and React.
Take into account that you'll need to properly "connect" ESLint and Prettier together in order for them not to be in conflict with each other.

File: webpack.config.prod.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+    filename: '[name].bundle.js',
+    chunkFilename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
+  resolve: {
+    extensions: ['.js', '.jsx', '.json'],
+    modules: [path.resolve(__dirname, './src'), 'node_modules'],
+  },
+  watch: true,
+  devServer: {
```
Comment (Dmitry-White) [inline webpack.config.prod.js]: Consider removing anything related to "development" from your "production" build

**Review by Dmitry-White — CHANGES_REQUESTED** [review]
Nicely done!

File: webpack.config.prod.js (hunk 2 above)
Comment (pixelpilot-dev) [reply webpack.config.prod.js]: fixed

File: package.json (hunk 1 above)
Comment (pixelpilot-dev) [reply package.json]: Thank you! I added stylelint, prettier and also updated eslint config.

**Review by Dmitry-White — APPROVED** [review]

**Pull request merged**